lib/logsource.h

```diff
@@ -29,7 +29,6 @@
 #include "stats/stats-registry.h"
 #include "window-size-counter.h"
 #include "dynamic-window.h"
-#include "ack-tracker/ack_tracker_types.h"
```

Comment:
For some reason I can't comment on `logproto-server.h`'s affected line... The same include removal can be done there, too.